http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/files/FileResource.java

```diff
@@ -68,20 +68,20 @@ public String uploadMultipart(@MultipartForm FileWrapper body) {
     @GET
     @Produces(MediaType.MULTIPART_FORM_DATA)
     @Path("/download-multipart")
-    @Blocking //https://github.com/quarkusio/quarkus/issues/25909
-    public Uni<FileWrapper> downloadMultipart() {
+    // @Blocking -> https://github.com/quarkusio/quarkus/issues/25909
+    public FileWrapper downloadMultipart() {
```

Comment:
synchronous methods default to worker threads so you don't need to return a "uni". 
Ref: https://quarkus.io/blog/resteasy-reactive-smart-dispatch/#new-world-new-rules